Python package set minor updates

mweinelt opened the pull request:
Regular python-unstable run with minor updates to libraries.

https://hydra.nixos.org/jobset/nixpkgs/python-unstable

fabaff committed:
python3Packages.cachetools: 4.2.1 -> 4.2.2

fabaff committed:
python3Packages.python-telegram-bot: add missing cachetools dependency

mweinelt committed:
python3Package.numpy: enable test on pypy

The bug in question was closed in 2019 and I saw no issue building
pypyPackages.numpy.

mweinelt committed:
python3Packages.SQLAlchemy-i18n: 1.0.3 -> 1.1.0

mweinelt committed:
python3Packages.sqlite-utils: 3.6 -> 3.9.1

mweinelt committed:
python3Packages.spyder-kernels: 1.10.2 -> 1.10.3

mweinelt committed:
python3Packages.srp: 1.0.17 -> 1.0.18

mweinelt committed:
python3Packages.sphinx_rtd_theme: 0.5.1 -> 0.5.2

mweinelt committed:
python3Packages.sunpy: 2.1.2 -> 2.1.5

mweinelt committed:
python3Packages.spyder: 4.2.4 -> 4.2.5

mweinelt committed:
python3Packages.sqlmap: 1.5.3 -> 1.5.6

mweinelt committed:
python3Packages.sphinxcontrib-tikz: 0.4.11 -> 0.4.13

mweinelt committed:
python3Packages.stripe: 2.56.0 -> 2.58.0

mweinelt committed:
python3Packages.tempora: 4.0.1 -> 4.0.2

mweinelt committed:
python3Packages.tld: 0.12.5 -> 0.12.6

mweinelt committed:
python3Packages.tifffile: 2021.3.17 -> 2021.6.14

mweinelt committed:
python3Packages.terminado: 0.9.3 -> 0.10.1

mweinelt committed:
python3Packages.towncrier: 19.2.0 -> 19.9.0

mweinelt committed:
python3Packages.tern: 2.5.0 -> 2.6.1

mweinelt committed:
python3Packages.svglib: 1.0.1 -> 1.1.0

mweinelt committed:
python3Packages.thespian: 3.10.3 -> 3.10.5

mweinelt committed:
python3Packages.tomlkit: 0.7.0 -> 0.7.2

mweinelt committed:
python3Packages.titlecase: 2.0.0 -> 2.2.0

mweinelt committed:
python3Packages.sympy: 1.7.1 -> 1.8

mweinelt committed:
python3Packages.testpath: 0.4.4 -> 0.5.0

mweinelt committed:
python3Packages.tubeup: 0.0.25 -> 0.0.26

mweinelt committed:
python3Packages.tqdm: 4.60.0 -> 4.61.1

mweinelt committed:
python3Packages.traitsui: 7.1.1 -> 7.2.1

mweinelt committed:
python3Packages.tvdb_api: 3.0.2 -> 3.1.0

mweinelt committed:
python3Packages.tox: 3.23.0 -> 3.23.1

mweinelt committed:
python3Packages.tvnamer: 3.0.2 -> 3.0.4

mweinelt committed:
python3Packages.treq: 21.1.0 -> 21.5.0

mweinelt committed:
python3Packages.trimesh: 3.9.10 -> 3.9.20

mweinelt committed:
python3Packages.transmission-rpc: 3.2.4 -> 3.2.5

mweinelt committed:
python3Packages.trytond: 5.8.5 -> 5.8.9

mweinelt committed:
python3Packages.trustme: 0.7.0 -> 0.8.0

mweinelt committed:
python3Packages.trio: 0.18.0 -> 0.19.0

mweinelt committed:
python3Packages.Twiggy: 0.5.0 -> 0.5.1

mweinelt committed:
python3Packages.PyVirtualDisplay: 2.1 -> 2.2

mweinelt committed:
python3Packages.typing_extensions: 3.7.4.3 -> 3.10.0.0

mweinelt committed:
python3Packages.vega: 3.4.0 -> 3.5.0

mweinelt committed:
python3Packages.vivisect: 0.1.0 -> 0.2.1

mweinelt committed:
python3Packages.typeguard: 2.12.0 -> 2.12.1

mweinelt committed:
python3Packages.typecode: 21.2.24 -> 21.6.1

mweinelt committed:
python3Packages.virtualenv: 20.4.3 -> 20.4.7

mweinelt committed:
python3Packages.twitter: 1.18.0 -> 1.19.3

mweinelt committed:
python3Packages.uritools: 3.0.1 -> 3.0.2

mweinelt committed:
python3Packages.typing: 3.7.4.3 -> 3.10.0.0

mweinelt committed:
python3Packages.yq: 2.12.0 -> 2.12.2

mweinelt committed:
python3Packages.xarray: 0.17.0 -> 0.18.2

mweinelt committed:
python3Packages.west: 0.10.1 -> 0.11.0

mweinelt committed:
python3Packages.xxhash: 2.0.0 -> 2.0.2

mweinelt committed:
python3Packages.whisper: 1.1.7 -> 1.1.8

mweinelt committed:
python3Packages.xml2rfc: 3.6.0 -> 3.8.0

mweinelt committed:
python3Packages.zarr: 2.6.1 -> 2.8.3

mweinelt committed:
python3Packages.vowpalwabbit: 8.9.0 -> 8.10.2

mweinelt committed:
python3Packages.xlsx2csv: 0.7.7 -> 0.7.8

mweinelt committed:
python3Packages.wcmatch: 8.1.2 -> 8.2

mweinelt committed:
python3Packages.XStatic-Pygments: 2.7.2.1 -> 2.9.0.1

mweinelt committed:
python3Packages.wurlitzer: 2.0.1 -> 2.1.0

mweinelt committed:
python3Packages.zstd: 1.4.8.1 -> 1.5.0.2

mweinelt committed:
python3Packages.zope.interface: 5.3.0 -> 5.4.0

mweinelt committed:
python3Packages.Kajiki: 0.8.2 -> 0.8.3

mweinelt committed:
python3Packages.azure-mgmt-apimanagement: 0.1.0 -> 0.3.0

Commit:
python3Packages.portpicker: fix build

Commit:
python3Packages.pydsdl: 1.9.4 -> 1.12.1

Commit:
python3Packages.pyroute2-ipset: 0.6.1 -> 0.6.4

Commit:
python3Packages.pytest-cov: fix build, add missing deps

Commit:
python3Packages.twitter: fix build, add missing deps

Commit:
python3Packages.testpath: fix source

Commit:
python3Packages.cligj: 0.5.0 -> 0.7.2

mweinelt committed:
python3Packages.fetchPypi: support wheels where dist & python differ

This introduces the `dist` argument to the `computeWheelUrl`, that
allows the distribution format in the path to differ from the python
version.

A wheel can be py3 and still have their file below the py2.py3 dist url.

mweinelt committed:
python3Packages.testpath: enable tests

Commit:
update-python-libraries: handle github branches and tags with same name

Commit:
python3Packages.aio-geojson-client: 0.14 -> 0.15

Commit:
python3Packages.aio-geojson-geonetnz-volcano: 0.5 -> 0.6

Commit:
python3Packages.amazon_kclpy: 2.0.1 -> 2.0.2

Commit:
python3Packages.aws-lambda-builders: 1.3.0 -> 1.4.0

Commit:
python3Packages.aiostream: 0.4.2 -> 0.4.3

Commit:
python3Packages.aiopvpc: 2.1.2 -> 2.2.0

Commit:
python3Packages.aio-georss-client: 0.7 -> 0.8

Commit:
python3Packages.ambee: 0.2.1 -> 0.3.0

Commit:
python3Packages.asyncwhois: 0.3.1 -> 0.3.2

Commit:
python3Packages.awslambdaric: 1.0.0 -> 1.1.1

Commit:
python3Packages.batchspawner: 1.0.1 -> 1.1.0

Commit:
python3Packages.certbot: 1.15.0 -> 1.16.0

Commit:
python3Packages.cfn-lint: 0.48.0 -> 0.51.0

Commit:
python3Packages.bip_utils: 1.9.0 -> 1.11.1

Commit:
python3Packages.chainer: 7.7.0 -> 7.8.0

Commit:
python3Packages.casbin: 0.19.1 -> 0.20.0

Commit:
python3Packages.deezer-python: 2.2.2 -> 2.2.4

Commit:
python3Packages.dnachisel: 3.2.6 -> 3.2.7

Commit:
python3Packages.dash: 1.19.0 -> 1.20.0

Commit:
python3Packages.django-logentry-admin: 1.0.4 -> 1.1.0

Commit:
python3Packages.diagrams: 0.19.1 -> 0.20.0

Commit:
python3Packages.construct: 2.10.63 -> 2.10.67

Commit:
python3Packages.django-anymail: 8.2 -> 8.4

Commit:
python3Packages.elementpath: 2.2.2 -> 2.2.3

Commit:
python3Packages.georss-ingv-centro-nazionale-terremoti-client: 0.4 -> 0.5

Commit:
python3Packages.enamlx: 0.4.3 -> 0.4.6

Commit:
python3Packages.exchangelib: 4.1.0 -> 4.4.0

jonringer committed:
python3Packages.georss-tfs-incidents-client: 0.2 -> 0.3

Commit:
python3Packages.georss-nrcan-earthquakes-client: 0.2 -> 0.3

jonringer committed:
python3Packages.gsd: 2.4.1 -> 2.4.2

jonringer committed:
python3Packages.gtts: 2.2.2 -> 2.2.3

jonringer committed:
python3Packages.h3: 3.7.2 -> 3.7.3

jonringer committed:
python3Packages.georss-wa-dfes-client: 0.2 -> 0.3

jonringer committed:
python3Packages.graphql-core: 3.1.4 -> 3.1.5

jonringer committed:
python3Packages.ibm-watson: 5.1.0 -> 5.2.0

jonringer committed:
python3Packages.ignite: 0.4.4.post1 -> 0.4.5

jonringer committed:
python3Packages.jc: 1.14.4 -> 1.15.4

jonringer committed:
python3Packages.inform: 1.23 -> 1.24

jonringer committed:
python3Packages.labelbox: 2.5.4 -> 2.6.0

jonringer committed:
python3Packages.libcst: 0.3.18 -> 0.3.19

jonringer committed:
python3Packages.kmapper: 2.0.0 -> 2.0.1

jonringer committed:
python3Packages.mongoengine: 0.23.0 -> 0.23.1

jonringer committed:
python3Packages.lsassy: 2.1.3 -> 2.1.5

jonringer committed:
python3Packages.mcstatus: 6.0.0 -> 6.1.0

jonringer committed:
python3Packages.mne-python: 0.22.0 -> 0.23.0

jonringer committed:
python3Packages.maestral: 1.4.3 -> 1.4.4

jonringer committed:
python3Packages.mkl-service: 2.3.0 -> 2.4.0

jonringer committed:
python3Packages.monty: 2021.3.3 -> 2021.6.10

jonringer committed:
python3Packages.pyatmo: 5.0.1 -> 5.1.0

jonringer committed:
python3Packages.perfplot: 0.8.10 -> 0.9.5

jonringer committed:
python3Packages.optuna: 2.6.0 -> 2.8.0

jonringer committed:
python3Packages.pyatv: 0.7.7 -> 0.8.0

jonringer committed:
python3Packages.pubnub: 5.1.1 -> 5.1.3

jonringer committed:
python3Packages.pycairo: 1.20.0 -> 1.20.1

jonringer committed:
python3Packages.pygccxml: 2.1.0 -> 2.2.1

jonringer committed:
python3Packages.pymemcache: 3.4.0 -> 3.4.4

jonringer committed:
python3Packages.pydantic: 1.8.1 -> 1.8.2

jonringer committed:
python3Packages.pynetdicom: 1.5.6 -> 1.5.7

jonringer committed:
python3Packages.pytest-ansible: 2.2.3 -> 2.2.4

jonringer committed:
python3Packages.pyvmomi: 7.0.1 -> 7.0.2

jonringer committed:
python3Packages.pytorch-metric-learning: 0.9.97 -> 0.9.99

jonringer committed:
python3Packages.pysaml2: 6.5.1 -> 6.5.2

jonringer committed:
python3Packages.pytube: 10.8.4 -> 10.8.5

jonringer committed:
python3Packages.rx: 3.1.1 -> 3.2.0

jonringer committed:
python3Packages.rq: 1.7.0 -> 1.8.1

jonringer committed:
python3Packages.robotframework: 4.0.1 -> 4.0.3

jonringer committed:
python3Packages.radish-bdd: 0.13.2 -> 0.13.4

jonringer committed:
python3Packages.sanic-testing: 0.4.0 -> 0.6.0

jonringer committed:
python3Packages.rich: 10.1.0 -> 10.4.0

jonringer committed:
python3Packages.sasmodels: 1.0.4 -> 1.0.5

jonringer committed:
python3Packages.simple-salesforce: 1.11.0 -> 1.11.2

jonringer committed:
python3Packages.screenlogicpy: 0.4.1 -> 0.4.2

jonringer committed:
python3Packages.scikit-hep-testdata: 0.4.3 -> 0.4.4

jonringer committed:
python3Packages.slack-sdk: 3.6.0 -> 3.7.0

jonringer committed:
python3Packages.soco: 0.22.3 -> 0.22.6

jonringer committed:
python3Packages.secure: 0.2.1 -> 0.3.0

jonringer committed:
python3Packages.sumo: 2.2.4 -> 2.2.5

jonringer committed:
python3Packages.starkbank-ecdsa: 1.1.0 -> 1.1.1

jonringer committed:
python3Packages.symengine: 0.7.0.post3 -> 0.7.2

jonringer committed:
python3Packages.tensorboardx: 2.1 -> 2.2

jonringer committed:
python3Packages.torchvision: 0.9.0 -> 0.10.0

jonringer committed:
python3Packages.termplotlib: 0.3.4 -> 0.3.5

jonringer committed:
python3Packages.trackpy: 0.4.2 -> 0.5.0

jonringer committed:
python3Packages.yara-python: 4.0.5 -> 4.1.0

jonringer committed:
python3Packages.sphinx: 3.5.4 -> 4.0.2

jonringer committed:
python3Packages.matplotlib-inline: init at 0.1.2

jonringer committed:
python3Packages.ipython: fix build, missing deps

jonringer committed:
python3Packages.sqlalchemy-utils: 0.36.6 -> 0.37.7

jonringer committed:
python3Packages.aiowinreg: fix build, add missing dep

r-rmcgibbo commented:
Result of `nixpkgs-review pr 127419` at 9c61dd94 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>267 packages marked as broken and skipped:</summary>
  <ul>
    <li>Literate</li>
    <li>OVMF-CSM</li>
    <li>acl2</li>
    <li>adoptopenjdk-hotspot-bin-13</li>
    <li>adoptopenjdk-hotspot-bin-14</li>
    <li>adoptopenjdk-jre-hotspot-bin-13</li>
    <li>adoptopenjdk-jre-hotspot-bin-14</li>
    <li>airtame</li>
    <li>ams-lv2</li>
    <li>apfelgrid</li>
    <li>applgrid</li>
    <li>artyFX</li>
    <li>asymptote</li>
    <li>autotrace</li>
    <li>beignet</li>
    <li>betaflight-configurator</li>
    <li>blastem</li>
    <li>bless</li>
    <li>bootiso</li>
    <li>bud</li>
    <li>cargo-rr</li>
    <li>cassandra_2_1</li>
    <li>cassandra_2_2</li>
    <li>chez</li>
    <li>clmagma</li>
    <li>colmapWithCuda</li>
    <li>cura_stable</li>
    <li>deeptools</li>
    <li>devede</li>
    <li>displaycal</li>
    <li>dtools</li>
    <li>dvd-slideshow</li>
    <li>electron_3</li>
    <li>electron_4</li>
    <li>electron_5</li>
    <li>embree</li>
    <li>embree2</li>
    <li>eterm</li>
    <li>evilvte</li>
    <li>exodus</li>
    <li>facter</li>
    <li>ffmpeg_2</li>
    <li>ffmpeg_2_8</li>
    <li>ffmpeg_3</li>
    <li>ffmpeg_3_4</li>
    <li>firestarter</li>
    <li>forge</li>
    <li>gb-backup</li>
    <li>git-dit</li>
    <li>gitin</li>
    <li>globalplatform</li>
    <li>globulation2</li>
    <li>gmtk</li>
    <li>gnome_mplayer</li>
    <li>gnuradio3_7</li>
    <li>gnuradio3_7Packages.ais</li>
    <li>gnuradio3_7Packages.gnuradio</li>
    <li>gnuradio3_7Packages.gsm</li>
    <li>gnuradio3_7Packages.limesdr</li>
    <li>gnuradio3_7Packages.nacl</li>
    <li>gnuradio3_7Packages.osmosdr</li>
    <li>gnuradio3_7Packages.rds</li>
    <li>go_1_14</li>
    <li>gogoclient</li>
    <li>gopro</li>
    <li>gppcscconnectionplugin</li>
    <li>gpshell</li>
    <li>grass</li>
    <li>green-pdfviewer</li>
    <li>gtkd</li>
    <li>gtmess</li>
    <li>hplipWithPlugin_3_16_11</li>
    <li>hplipWithPlugin_3_18_5</li>
    <li>hplip_3_16_11</li>
    <li>hplip_3_18_5</li>
    <li>idris2</li>
    <li>impressive</li>
    <li>intecture-cli</li>
    <li>intel-compute-runtime</li>
    <li>intel-media-driver</li>
    <li>invoice2data</li>
    <li>isabelle</li>
    <li>jboss</li>
    <li>jfbpdf</li>
    <li>jfbview</li>
    <li>k2pdfopt</li>
    <li>k3d</li>
    <li>keyfinder</li>
    <li>libav</li>
    <li>libav_0_8</li>
    <li>libav_11</li>
    <li>libav_12</li>
    <li>libgroove</li>
    <li>libvirt_5_9_0</li>
    <li>libvmi</li>
    <li>linuxPackages_hardkernel_4_14.akvcam</li>
    <li>linuxPackages_hardkernel_4_14.bcc</li>
    <li>linuxPackages_hardkernel_4_14.evdi</li>
    <li>linuxPackages_hardkernel_4_14.openrazer</li>
    <li>linuxPackages_hardkernel_4_14.systemtap</li>
    <li>linuxPackages_hardkernel_4_14.xpadneo</li>
    <li>linuxPackages_hardkernel_4_14.zfs</li>
    <li>linuxPackages_hardkernel_4_14.zfsStable</li>
    <li>linuxPackages_hardkernel_4_14.zfsUnstable</li>
    <li>linuxPackages_hardkernel_latest.akvcam</li>
    <li>linuxPackages_hardkernel_latest.bcc</li>
    <li>linuxPackages_hardkernel_latest.evdi</li>
    <li>linuxPackages_hardkernel_latest.openrazer</li>
    <li>linuxPackages_hardkernel_latest.systemtap</li>
    <li>linuxPackages_hardkernel_latest.xpadneo</li>
    <li>linuxPackages_hardkernel_latest.zfs</li>
    <li>linuxPackages_hardkernel_latest.zfsStable</li>
    <li>linuxPackages_hardkernel_latest.zfsUnstable</li>
    <li>linuxPackages_testing_bcachefs.akvcam</li>
    <li>linuxPackages_testing_bcachefs.bcc</li>
    <li>linuxPackages_testing_bcachefs.evdi</li>
    <li>linuxPackages_testing_bcachefs.oci-seccomp-bpf-hook</li>
    <li>linuxPackages_testing_bcachefs.openrazer</li>
    <li>linuxPackages_testing_bcachefs.systemtap</li>
    <li>linuxPackages_testing_bcachefs.xpadneo</li>
    <li>linuxPackages_testing_bcachefs.zfs</li>
    <li>linuxPackages_testing_bcachefs.zfsStable</li>
    <li>linuxPackages_testing_bcachefs.zfsUnstable</li>
    <li>loxodo</li>
    <li>luxcorerender</li>
    <li>magma</li>
    <li>mame</li>
    <li>meo</li>
    <li>metamorphose2</li>
    <li>mpich</li>
    <li>mrxvt</li>
    <li>mupdf_1_17</li>
    <li>mvapich</li>
    <li>natron</li>
    <li>nixui</li>
    <li>nomad_0_12</li>
    <li>nvtop</li>
    <li>ocropus</li>
    <li>odpdown</li>
    <li>odroid-xu3-bootloader</li>
    <li>onedrive</li>
    <li>openems</li>
    <li>openrw</li>
    <li>openssh_gssapi</li>
    <li>openssh_hpn</li>
    <li>opentsdb</li>
    <li>pcsxr</li>
    <li>pdfdiff</li>
    <li>pgadmin</li>
    <li>photoflow</li>
    <li>pig</li>
    <li>plover.stable</li>
    <li>poedit</li>
    <li>polymake</li>
    <li>pony-stable</li>
    <li>processing</li>
    <li>pulseeffects-legacy</li>
    <li>pulseeffects-pw</li>
    <li>python38Packages.HTSeq</li>
    <li>python38Packages.bwapy</li>
    <li>python38Packages.caffe</li>
    <li>python38Packages.cnvkit</li>
    <li>python38Packages.dask-gateway</li>
    <li>python38Packages.dask-jobqueue</li>
    <li>python38Packages.dask-ml</li>
    <li>python38Packages.dask-mpi</li>
    <li>python38Packages.dask-xgboost</li>
    <li>python38Packages.datashader</li>
    <li>python38Packages.fipy</li>
    <li>python38Packages.foundationdb51</li>
    <li>python38Packages.foundationdb52</li>
    <li>python38Packages.foundationdb60</li>
    <li>python38Packages.foundationdb61</li>
    <li>python38Packages.foxdot</li>
    <li>python38Packages.ignite</li>
    <li>python38Packages.mkl-service</li>
    <li>python38Packages.pycuda</li>
    <li>python38Packages.pygmt</li>
    <li>python38Packages.pymystem3</li>
    <li>python38Packages.pynvml</li>
    <li>python38Packages.python-csxcad</li>
    <li>python38Packages.python-openems</li>
    <li>python38Packages.pythonnet</li>
    <li>python38Packages.pytorchWithCuda</li>
    <li>python38Packages.spacy</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.streamz</li>
    <li>python38Packages.stumpy</li>
    <li>python38Packages.tensorflowWithCuda</li>
    <li>python38Packages.textacy</li>
    <li>python38Packages.thinc</li>
    <li>python39Packages.HTSeq</li>
    <li>python39Packages.bwapy</li>
    <li>python39Packages.caffe</li>
    <li>python39Packages.cnvkit</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-jobqueue</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.dask-xgboost</li>
    <li>python39Packages.fipy</li>
    <li>python39Packages.foundationdb51</li>
    <li>python39Packages.foundationdb52</li>
    <li>python39Packages.foundationdb60</li>
    <li>python39Packages.foundationdb61</li>
    <li>python39Packages.foxdot</li>
    <li>python39Packages.ignite</li>
    <li>python39Packages.mkl-service</li>
    <li>python39Packages.pycuda</li>
    <li>python39Packages.pygmt</li>
    <li>python39Packages.pymystem3</li>
    <li>python39Packages.pynvml</li>
    <li>python39Packages.python-csxcad</li>
    <li>python39Packages.python-openems</li>
    <li>python39Packages.pythonnet</li>
    <li>python39Packages.pytorchWithCuda</li>
    <li>python39Packages.streamz</li>
    <li>python39Packages.tensorflowWithCuda</li>
    <li>python39Packages.thinc</li>
    <li>qemu_xen</li>
    <li>qemu_xen-light</li>
    <li>qemu_xen_4_10</li>
    <li>qemu_xen_4_10-light</li>
    <li>qlandkartegt</li>
    <li>qubes-core-vchan-xen</li>
    <li>rainbowstream</li>
    <li>ring-daemon</li>
    <li>rocm-opencl-icd</li>
    <li>rocm-opencl-runtime</li>
    <li>root5</li>
    <li>rund</li>
    <li>rxvt</li>
    <li>scribus</li>
    <li>secretscanner</li>
    <li>shellinabox</li>
    <li>shotgun</li>
    <li>sonic-pi</li>
    <li>spark</li>
    <li>split2flac</li>
    <li>spotifywm</li>
    <li>spring</li>
    <li>springLobby</li>
    <li>stepmania</li>
    <li>swift</li>
    <li>termplay</li>
    <li>thrift-0_10</li>
    <li>tilix</li>
    <li>tlaps</li>
    <li>torchat</li>
    <li>truvari</li>
    <li>tvheadend</li>
    <li>twinkle</li>
    <li>unetbootin</li>
    <li>unicap</li>
    <li>vbam</li>
    <li>vcs</li>
    <li>vp</li>
    <li>wimlib</li>
    <li>winpdb</li>
    <li>woeusb</li>
    <li>xfitter</li>
    <li>xorg.libAppleWM</li>
    <li>xpdf</li>
    <li>xsw</li>
    <li>yaxg</li>
    <li>yosys-bluespec</li>
    <li>zandronum</li>
    <li>zgv</li>
  </ul>
</details>
<details>
  <summary>10 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/1eebd88a34594723bc71e07f6e76b051">jc</a> (python38Packages.jc)</li>
    <li><a href="https://gist.github.com/eb8b7c7cad55d9c9156716bb44a14a60">python38Packages.Kajiki</a></li>
    <li><a href="https://gist.github.com/b05d8069cb5c62a7b9d03507771d2bb7">python38Packages.atsim_potentials</a></li>
    <li><a href="https://gist.github.com/f5dbc7429573d7427366da75a4b344b5">python38Packages.cerberus</a></li>
    <li><a href="https://gist.github.com/9cf76992aa8162ff4c5b4c35e4d8d997">python38Packages.email_validator</a></li>
    <li><a href="https://gist.github.com/3e5750ecb8fc19b52204bde840531e68">python38Packages.mwparserfromhell</a></li>
    <li><a href="https://gist.github.com/c801a10890248717ff2bb548dadc0a94">python38Packages.panflute</a></li>
    <li><a href="https://gist.github.com/6989dbdcc8eecc8127a527c34d595417">python38Packages.whisper</a></li>
    <li><a href="https://gist.github.com/8542f0784531b7bfd0135d18de78f2cf">python39Packages.mwparserfromhell</a></li>
    <li><a href="https://gist.github.com/fda12d3406769f31737a04d3d64fbfee">python39Packages.whisper</a></li>
  </ul>
</details>
<details>
  <summary>23420 packages skipped due to time constraints:</summary>
  <ul>
    <li>AusweisApp2</li>
    <li>CHOWTapeModel</li>
    <li>DisnixWebService</li>
    <li>EBTKS</li>
    <li>EmptyEpsilon</li>
    <li>Fabric (python38Packages.Fabric)</li>
    <li>MACS2</li>
    <li>MIDIVisualizer</li>
    <li>MMA</li>
    <li>OVMF</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>478 packages built successfully:</summary>
  <ul>
    <li>docutils (python38Packages.docutils)</li>
    <li>fribidi</li>
    <li>gi-docgen</li>
    <li>gtk-doc (gnome2.gtkdoc)</li>
    <li>humanfriendly (python38Packages.humanfriendly)</li>
    <li>jpylyzer (python38Packages.jpylyzer)</li>
    <li>libglvnd</li>
    <li>libsysprof-capture</li>
    <li>libvdpau</li>
    <li>meson</li>
    <li>python38Packages.Babel</li>
    <li>python38Packages.EasyProcess</li>
    <li>python38Packages.PyStemmer</li>
    <li>python38Packages.absl-py</li>
    <li>python38Packages.aiofiles</li>
    <li>python38Packages.aiostream</li>
    <li>python38Packages.aiowinreg</li>
    <li>python38Packages.alabaster</li>
    <li>python38Packages.amqp</li>
    <li>python38Packages.ansicolor</li>
    <li>python38Packages.apipkg</li>
    <li>python38Packages.appdirs</li>
    <li>python38Packages.argh</li>
    <li>python38Packages.arpeggio</li>
    <li>python38Packages.arrow</li>
    <li>python38Packages.async-timeout</li>
    <li>python38Packages.asyncio-dgram</li>
    <li>python38Packages.asyncio-mqtt</li>
    <li>python38Packages.atomicwrites</li>
    <li>python38Packages.atpublic</li>
    <li>python38Packages.attrs</li>
    <li>python38Packages.awscrt</li>
    <li>python38Packages.backcall</li>
    <li>python38Packages.bids-validator</li>
    <li>python38Packages.bip_utils</li>
    <li>python38Packages.bitarray</li>
    <li>python38Packages.blessed</li>
    <li>python38Packages.boto3</li>
    <li>python38Packages.botocore</li>
    <li>python38Packages.bracex</li>
    <li>python38Packages.brotli</li>
    <li>python38Packages.bsdiff4</li>
    <li>python38Packages.bugsnag</li>
    <li>python38Packages.case</li>
    <li>python38Packages.cbor2</li>
    <li>python38Packages.certifi</li>
    <li>python38Packages.cfgv</li>
    <li>python38Packages.cheetah3</li>
    <li>python38Packages.ciso8601</li>
    <li>python38Packages.citeproc-py</li>
    <li>python38Packages.click</li>
    <li>python38Packages.cligj</li>
    <li>python38Packages.cloudpickle</li>
    <li>python38Packages.cock</li>
    <li>python38Packages.colorama</li>
    <li>python38Packages.configparser</li>
    <li>python38Packages.coverage</li>
    <li>python38Packages.cppy</li>
    <li>python38Packages.cucumber-tag-expressions</li>
    <li>python38Packages.cx_Freeze</li>
    <li>python38Packages.cycler</li>
    <li>python38Packages.cython</li>
    <li>python38Packages.dash-core-components</li>
    <li>python38Packages.dash-html-components</li>
    <li>python38Packages.dash-renderer</li>
    <li>python38Packages.dash-table</li>
    <li>python38Packages.dataclasses-json</li>
    <li>python38Packages.dateutil (python38Packages.python-dateutil)</li>
    <li>python38Packages.decopatch</li>
    <li>python38Packages.distlib</li>
    <li>python38Packages.django (python38Packages.django_2)</li>
    <li>python38Packages.django-cleanup</li>
    <li>python38Packages.django-logentry-admin</li>
    <li>python38Packages.django-picklefield</li>
    <li>python38Packages.django-q</li>
    <li>python38Packages.django_appconf</li>
    <li>python38Packages.django_compressor</li>
    <li>python38Packages.django_environ</li>
    <li>python38Packages.django_guardian</li>
    <li>python38Packages.djangoql</li>
    <li>python38Packages.dns (python38Packages.dnspython)</li>
    <li>python38Packages.dnslib</li>
    <li>python38Packages.dockerfile-parse</li>
    <li>python38Packages.dpkt</li>
    <li>python38Packages.ecdsa</li>
    <li>python38Packages.elementpath</li>
    <li>python38Packages.entrypoint2</li>
    <li>python38Packages.eradicate</li>
    <li>python38Packages.et_xmlfile</li>
    <li>python38Packages.eventlet</li>
    <li>python38Packages.execnet</li>
    <li>python38Packages.fastimport</li>
    <li>python38Packages.feedparser</li>
    <li>python38Packages.filelock</li>
    <li>python38Packages.flake8</li>
    <li>python38Packages.flaky</li>
    <li>python38Packages.freezegun</li>
    <li>python38Packages.funcy</li>
    <li>python38Packages.future</li>
    <li>python38Packages.genshi</li>
    <li>python38Packages.gitdb</li>
    <li>python38Packages.gorilla</li>
    <li>python38Packages.graphql-core</li>
    <li>python38Packages.green</li>
    <li>python38Packages.greenlet</li>
    <li>python38Packages.gunicorn</li>
    <li>python38Packages.hijri-converter</li>
    <li>python38Packages.httptools</li>
    <li>python38Packages.humanize</li>
    <li>python38Packages.hyperframe</li>
    <li>python38Packages.idna</li>
    <li>python38Packages.imagesize</li>
    <li>python38Packages.importlib-metadata</li>
    <li>python38Packages.influxdb-client</li>
    <li>python38Packages.iniconfig</li>
    <li>python38Packages.intreehooks</li>
    <li>python38Packages.iocapture</li>
    <li>python38Packages.ipython_genutils</li>
    <li>python38Packages.iso8601</li>
    <li>python38Packages.itsdangerous</li>
    <li>python38Packages.jdcal</li>
    <li>python38Packages.jinja2</li>
    <li>python38Packages.jmespath</li>
    <li>python38Packages.jsondiff</li>
    <li>python38Packages.libevdev</li>
    <li>python38Packages.libtmux</li>
    <li>python38Packages.linecache2</li>
    <li>python38Packages.lxml</li>
    <li>python38Packages.makefun</li>
    <li>python38Packages.manhole</li>
    <li>python38Packages.markdown</li>
    <li>python38Packages.markupsafe</li>
    <li>python38Packages.marshmallow</li>
    <li>python38Packages.marshmallow-enum</li>
    <li>python38Packages.mccabe</li>
    <li>python38Packages.mcstatus</li>
    <li>python38Packages.mock</li>
    <li>python38Packages.mongoengine</li>
    <li>python38Packages.monotonic</li>
    <li>python38Packages.more-itertools</li>
    <li>python38Packages.mypy-boto3-s3</li>
    <li>python38Packages.mypy-extensions</li>
    <li>python38Packages.nclib</li>
    <li>python38Packages.netifaces</li>
    <li>python38Packages.nine</li>
    <li>python38Packages.nodeenv</li>
    <li>python38Packages.nose</li>
    <li>python38Packages.nosexcover</li>
    <li>python38Packages.ntplib</li>
    <li>python38Packages.nunavut</li>
    <li>python38Packages.olefile</li>
    <li>python38Packages.openpyxl</li>
    <li>python38Packages.ordereddict</li>
    <li>python38Packages.packaging</li>
    <li>python38Packages.paho-mqtt</li>
    <li>python38Packages.parts</li>
    <li>python38Packages.pathpy</li>
    <li>python38Packages.pathtools</li>
    <li>python38Packages.patiencediff</li>
    <li>python38Packages.pbr</li>
    <li>python38Packages.pex</li>
    <li>python38Packages.pexpect</li>
    <li>python38Packages.pickleshare</li>
    <li>python38Packages.pip</li>
    <li>python38Packages.pkgconfig</li>
    <li>python38Packages.plone-testing</li>
    <li>python38Packages.pluggy</li>
    <li>python38Packages.pluginbase</li>
    <li>python38Packages.ply</li>
    <li>python38Packages.polib</li>
    <li>python38Packages.portpicker</li>
    <li>python38Packages.pretend</li>
    <li>python38Packages.process-tests</li>
    <li>python38Packages.prometheus_client</li>
    <li>python38Packages.psutil</li>
    <li>python38Packages.ptyprocess</li>
    <li>python38Packages.py</li>
    <li>python38Packages.py-cpuinfo</li>
    <li>python38Packages.py_scrypt</li>
    <li>python38Packages.pyasn1</li>
    <li>python38Packages.pybindgen</li>
    <li>python38Packages.pycodestyle</li>
    <li>python38Packages.pycparser</li>
    <li>python38Packages.pydocstyle</li>
    <li>python38Packages.pydsdl</li>
    <li>python38Packages.pyflakes</li>
    <li>python38Packages.pygments</li>
    <li>python38Packages.pylama</li>
    <li>python38Packages.pymediainfo</li>
    <li>python38Packages.pymemcache</li>
    <li>python38Packages.pymongo</li>
    <li>python38Packages.pyparsing</li>
    <li>python38Packages.pypinyin</li>
    <li>python38Packages.pyroute2-core</li>
    <li>python38Packages.pyroute2-ipset</li>
    <li>python38Packages.pysha3</li>
    <li>python38Packages.pysigset</li>
    <li>python38Packages.pysmb</li>
    <li>python38Packages.pysocks</li>
    <li>python38Packages.pysqlcipher3</li>
    <li>python38Packages.pytest (python38Packages.pytest_6)</li>
    <li>python38Packages.pytest-asyncio</li>
    <li>python38Packages.pytest-benchmark</li>
    <li>python38Packages.pytest-cases</li>
    <li>python38Packages.pytest-celery</li>
    <li>python38Packages.pytestcov (python38Packages.pytest-cov)</li>
    <li>python38Packages.pytest-django</li>
    <li>python38Packages.pytest-expect</li>
    <li>python38Packages.pytest-flake8</li>
    <li>python38Packages.pytest-forked</li>
    <li>python38Packages.pytest-freezegun</li>
    <li>python38Packages.pytest-helpers-namespace</li>
    <li>python38Packages.pytest-html</li>
    <li>python38Packages.pytest-metadata</li>
    <li>python38Packages.pytest-mock</li>
    <li>python38Packages.pytestrunner (python38Packages.pytest-runner)</li>
    <li>python38Packages.pytest-snapshot</li>
    <li>python38Packages.pytest-subtests</li>
    <li>python38Packages.pytest-testmon</li>
    <li>python38Packages.pytest-xprocess</li>
    <li>python38Packages.python-sql</li>
    <li>python38Packages.python_magic</li>
    <li>python38Packages.pytoml</li>
    <li>python38Packages.pytz</li>
    <li>python38Packages.pyyaml</li>
    <li>python38Packages.queuelib</li>
    <li>python38Packages.rcssmin</li>
    <li>python38Packages.redis</li>
    <li>python38Packages.regex</li>
    <li>python38Packages.relatorio</li>
    <li>python38Packages.rfc3986</li>
    <li>python38Packages.rjsmin</li>
    <li>python38Packages.rnc2rng</li>
    <li>python38Packages.rply</li>
    <li>python38Packages.rq</li>
    <li>python38Packages.ruamel-base (python38Packages.ruamel_base)</li>
    <li>python38Packages.ruamel-yaml (python38Packages.ruamel_yaml)</li>
    <li>python38Packages.ruamel-yaml-clib (python38Packages.ruamel_yaml_clib)</li>
    <li>python38Packages.rx</li>
    <li>python38Packages.s3transfer</li>
    <li>python38Packages.schedule</li>
    <li>python38Packages.screenlogicpy</li>
    <li>python38Packages.semantic-version</li>
    <li>python38Packages.serpent</li>
    <li>python38Packages.setuptools</li>
    <li>python38Packages.setuptools-declarative-requirements</li>
    <li>python38Packages.setuptools-rust</li>
    <li>python38Packages.setuptools-scm</li>
    <li>python38Packages.sgmllib3k</li>
    <li>python38Packages.simplejson</li>
    <li>python38Packages.six</li>
    <li>python38Packages.smartypants</li>
    <li>python38Packages.smmap</li>
    <li>python38Packages.snowballstemmer</li>
    <li>python38Packages.sortedcontainers</li>
    <li>python38Packages.sphinxcontrib-applehelp</li>
    <li>python38Packages.sphinxcontrib-devhelp</li>
    <li>python38Packages.sphinxcontrib-htmlhelp</li>
    <li>python38Packages.sphinxcontrib-jsmath</li>
    <li>python38Packages.sphinxcontrib-qthelp</li>
    <li>python38Packages.sphinxcontrib-serializinghtml</li>
    <li>python38Packages.sphinxcontrib-websupport</li>
    <li>sqlmap (python38Packages.sqlmap)</li>
    <li>python38Packages.sqlparse</li>
    <li>python38Packages.srp</li>
    <li>python38Packages.starkbank-ecdsa</li>
    <li>python38Packages.stringcase</li>
    <li>python38Packages.sybil</li>
    <li>python38Packages.tappy</li>
    <li>python38Packages.terminado</li>
    <li>python38Packages.termstyle</li>
    <li>python38Packages.testpath</li>
    <li>python38Packages.thespian</li>
    <li>python38Packages.titlecase</li>
    <li>python38Packages.toml</li>
    <li>python38Packages.tomlkit</li>
    <li>python38Packages.tornado</li>
    <li>python38Packages.traceback2</li>
    <li>python38Packages.twiggy</li>
    <li>python38Packages.twitter</li>
    <li>python38Packages.typeguard</li>
    <li>python38Packages.typing-extensions</li>
    <li>python38Packages.typing-inspect</li>
    <li>python38Packages.typogrify</li>
    <li>python38Packages.u-msgpack-python</li>
    <li>python38Packages.ukpostcodeparser</li>
    <li>python38Packages.unidecode</li>
    <li>python38Packages.unittest2</li>
    <li>python38Packages.uritools</li>
    <li>python38Packages.urllib3</li>
    <li>python38Packages.vine</li>
    <li>python38Packages.virtual-display</li>
    <li>python38Packages.wcmatch</li>
    <li>python38Packages.wcwidth</li>
    <li>python38Packages.webencodings</li>
    <li>python38Packages.webob</li>
    <li>python38Packages.wheel</li>
    <li>python38Packages.winacl</li>
    <li>python38Packages.wrapt</li>
    <li>python38Packages.wurlitzer</li>
    <li>xlsx2csv (python38Packages.xlsx2csv)</li>
    <li>python38Packages.xmltodict</li>
    <li>python38Packages.xstatic-pygments</li>
    <li>python38Packages.xxhash</li>
    <li>python38Packages.yara-python</li>
    <li>python38Packages.zipp</li>
    <li>python38Packages.zope_event</li>
    <li>python38Packages.zope_exceptions</li>
    <li>python38Packages.zope_interface</li>
    <li>python38Packages.zope_location</li>
    <li>python38Packages.zope_proxy</li>
    <li>python38Packages.zope_testing</li>
    <li>python38Packages.zstd</li>
    <li>python39Packages.EasyProcess</li>
    <li>python39Packages.absl-py</li>
    <li>python39Packages.aiofiles</li>
    <li>python39Packages.aiowinreg</li>
    <li>python39Packages.alabaster</li>
    <li>python39Packages.ansicolor</li>
    <li>python39Packages.appdirs</li>
    <li>python39Packages.async-timeout</li>
    <li>python39Packages.atomicwrites</li>
    <li>python39Packages.attrs</li>
    <li>python39Packages.awscrt</li>
    <li>python39Packages.bids-validator</li>
    <li>python39Packages.bip_utils</li>
    <li>python39Packages.bitarray</li>
    <li>python39Packages.bsdiff4</li>
    <li>python39Packages.cfgv</li>
    <li>python39Packages.cheetah3</li>
    <li>python39Packages.colorama</li>
    <li>python39Packages.coverage</li>
    <li>python39Packages.cppy</li>
    <li>python39Packages.cx_Freeze</li>
    <li>python39Packages.cycler</li>
    <li>python39Packages.dash-core-components</li>
    <li>python39Packages.dash-html-components</li>
    <li>python39Packages.dash-renderer</li>
    <li>python39Packages.dash-table</li>
    <li>python39Packages.dateutil (python39Packages.python-dateutil)</li>
    <li>python39Packages.distlib</li>
    <li>python39Packages.dns (python39Packages.dnspython)</li>
    <li>python39Packages.dnslib</li>
    <li>python39Packages.docutils</li>
    <li>python39Packages.dpkt</li>
    <li>python39Packages.ecdsa</li>
    <li>python39Packages.elementpath</li>
    <li>python39Packages.eventlet</li>
    <li>python39Packages.fastimport</li>
    <li>python39Packages.feedparser</li>
    <li>python39Packages.funcy</li>
    <li>python39Packages.gorilla</li>
    <li>python39Packages.greenlet</li>
    <li>python39Packages.httptools</li>
    <li>python39Packages.humanfriendly</li>
    <li>python39Packages.imagesize</li>
    <li>python39Packages.importlib-metadata</li>
    <li>python39Packages.iniconfig</li>
    <li>python39Packages.iocapture</li>
    <li>python39Packages.ipython_genutils</li>
    <li>python39Packages.jsondiff</li>
    <li>python39Packages.libtmux</li>
    <li>python39Packages.linecache2</li>
    <li>python39Packages.makefun</li>
    <li>python39Packages.manhole</li>
    <li>python39Packages.mock</li>
    <li>python39Packages.mongoengine</li>
    <li>python39Packages.monotonic</li>
    <li>python39Packages.more-itertools</li>
    <li>python39Packages.nclib</li>
    <li>python39Packages.netifaces</li>
    <li>python39Packages.nodeenv</li>
    <li>python39Packages.nose</li>
    <li>python39Packages.ntplib</li>
    <li>python39Packages.nunavut</li>
    <li>python39Packages.olefile</li>
    <li>python39Packages.parts</li>
    <li>python39Packages.patiencediff</li>
    <li>python39Packages.pbr</li>
    <li>python39Packages.pex</li>
    <li>python39Packages.pexpect</li>
    <li>python39Packages.pip</li>
    <li>python39Packages.pkgconfig</li>
    <li>python39Packages.plone-testing</li>
    <li>python39Packages.pluggy</li>
    <li>python39Packages.polib</li>
    <li>python39Packages.portpicker</li>
    <li>python39Packages.pretend</li>
    <li>python39Packages.process-tests</li>
    <li>python39Packages.prometheus_client</li>
    <li>python39Packages.psutil</li>
    <li>python39Packages.ptyprocess</li>
    <li>python39Packages.py</li>
    <li>python39Packages.py_scrypt</li>
    <li>python39Packages.pyasn1</li>
    <li>python39Packages.pybindgen</li>
    <li>python39Packages.pycparser</li>
    <li>python39Packages.pydsdl</li>
    <li>python39Packages.pyflakes</li>
    <li>python39Packages.pygments</li>
    <li>python39Packages.pymongo</li>
    <li>python39Packages.pyroute2-core</li>
    <li>python39Packages.pyroute2-ipset</li>
    <li>python39Packages.pysha3</li>
    <li>python39Packages.pysigset</li>
    <li>python39Packages.pysmb</li>
    <li>python39Packages.pysocks</li>
    <li>python39Packages.pytest-celery</li>
    <li>python39Packages.python-sql</li>
    <li>python39Packages.pytz</li>
    <li>python39Packages.regex</li>
    <li>python39Packages.ruamel-base (python39Packages.ruamel_base)</li>
    <li>python39Packages.ruamel-yaml (python39Packages.ruamel_yaml)</li>
    <li>python39Packages.ruamel-yaml-clib (python39Packages.ruamel_yaml_clib)</li>
    <li>python39Packages.rx</li>
    <li>python39Packages.schedule</li>
    <li>python39Packages.semantic-version</li>
    <li>python39Packages.serpent</li>
    <li>python39Packages.setuptools</li>
    <li>python39Packages.setuptools-rust</li>
    <li>python39Packages.setuptools-scm</li>
    <li>python39Packages.sgmllib3k</li>
    <li>python39Packages.six</li>
    <li>python39Packages.sortedcontainers</li>
    <li>python39Packages.sphinxcontrib-applehelp</li>
    <li>python39Packages.sphinxcontrib-devhelp</li>
    <li>python39Packages.sphinxcontrib-htmlhelp</li>
    <li>python39Packages.sphinxcontrib-jsmath</li>
    <li>python39Packages.sphinxcontrib-qthelp</li>
    <li>python39Packages.sphinxcontrib-serializinghtml</li>
    <li>python39Packages.sphinxcontrib-websupport</li>
    <li>python39Packages.sqlmap</li>
    <li>python39Packages.srp</li>
    <li>python39Packages.terminado</li>
    <li>python39Packages.thespian</li>
    <li>python39Packages.titlecase</li>
    <li>python39Packages.toml</li>
    <li>python39Packages.tomlkit</li>
    <li>python39Packages.tornado</li>
    <li>python39Packages.traceback2</li>
    <li>python39Packages.twiggy</li>
    <li>python39Packages.twitter</li>
    <li>python39Packages.typing-extensions</li>
    <li>python39Packages.u-msgpack-python</li>
    <li>python39Packages.uritools</li>
    <li>python39Packages.virtual-display</li>
    <li>python39Packages.wcwidth</li>
    <li>python39Packages.wheel</li>
    <li>python39Packages.winacl</li>
    <li>python39Packages.xlsx2csv</li>
    <li>python39Packages.xstatic-pygments</li>
    <li>python39Packages.xxhash</li>
    <li>python39Packages.zipp</li>
    <li>python39Packages.zope_event</li>
    <li>python39Packages.zope_exceptions</li>
    <li>python39Packages.zope_interface</li>
    <li>python39Packages.zope_location</li>
    <li>python39Packages.zope_proxy</li>
    <li>python39Packages.zope_testing</li>
    <li>xkeyboard_config (xorg.xkeyboardconfig)</li>
    <li>xorg.libICE</li>
    <li>xorg.libSM</li>
    <li>xorg.libX11</li>
    <li>xorg.libXau</li>
    <li>xorg.libXcomposite</li>
    <li>xorg.libXcursor</li>
    <li>xorg.libXdmcp</li>
    <li>xorg.libXext</li>
    <li>xorg.libXfixes</li>
    <li>xorg.libXinerama</li>
    <li>xorg.libXrandr</li>
    <li>xorg.libXrender</li>
    <li>xorg.libXt</li>
    <li>xorg.libxcb</li>
    <li>xorg.xcbutil</li>
    <li>xorg.xcbutilkeysyms</li>
    <li>xorg.xcbutilwm</li>
    <li>xorg.xorgproto</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

mweinelt committed:
python3Packages.Cerberus: fix build

The pypi sources don't contain everything required to run tests.

mweinelt committed:
python3Packages.email-validator: fix build, enable tests

dotlambda committed:
python3Packages.h2: fix tests

mweinelt committed:
python3Packages.h2: clean up

mweinelt committed:
python2Packages.ruamel-yaml: reinit at 0.16.13

mweinelt committed:
python3Packages.google-auth: add missing py2uf dependency

dotlambda committed:
python3Packages.watchdog: re-run flaky tests

And get rid of `PytestUnknownMarkWarning: Unknown pytest.mark.flaky`.

dotlambda committed:
pythonPackages.env: make compatible with click 8

Also clean up inputs and define checkPhase.

mweinelt committed:
python3Packages.pyroute2-core: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-ethtool: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-ipdb: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-ndb: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-nftables: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-nslink: 0.6.1 -> 0.6.4

mweinelt committed:
python3Packages.pyroute2-protocols: 0.6.1 -> 0.6.4

dotlambda committed:
borgbackup: make compatible with sphinx 4

mweinelt committed:
python3Packages.aiopvpc: fix build; add missing holidays dependency

mweinelt committed:
python3Packages.jsonrpc-base: 2.0.0 -> 2.1.0

mweinelt committed:
python3Packages.pyatmo: fix build; add missing pytest-mock dep

mweinelt committed:
python3Packages.surepy: add attrs dep, relax attrs and click dep

mweinelt committed:
python3Packages.uvicorn: 0.13.2 -> 0.14.0

Separates tests out into a separate attribute to resolve infinite
recursion problems with httpx/httpcore.

jonringer committed:
azure-cli: fix build

jonringer committed:
python3Packages.fsspec: 2021.05.0 -> 2021.06.0

jonringer committed:
python3Packages.snowflake-connector-python: fix build

jonringer committed:
python3Packages.gpyopt: unstable-2019-09-25 -> 1.2.6

jonringer committed:
python3Packages.jwcrypto: fix build, add missing dep

jonringer committed:
python3Packages.auth0-python: fix build, add missing dep

mweinelt committed:
python3Packages.async-upnp-client: 0.18.0 -> 0.19.0

mweinelt committed:
matrix-synapse-tools.synadm: relax click constraints

mweinelt committed:
python3Packages.sqlalchemy-imageattach: mark as broken

This broke, when the declarative API of SQLAlchemy changed in the 1.4.
release series.

The library seems otherwise unmaintained since 2017.

https://github.com/dahlia/sqlalchemy-imageattach/issues/47

dotlambda committed:
pythonPackages.pybind11: improve expression

We shouldn't build and install the package in preFixup.

dotlambda committed:
python3Packages.moto relax MarkupSafe constraint

dotlambda committed:
pythonPackages.click-default-group: fix tests

Also use pytestCheckHook and pythonImportsCheck.

dotlambda committed:
python3Packages.miniaudio: init at 1.44

dotlambda committed:
python3Packages.pyatv: update dependencies

dotlambda committed:
pythonPackages.pyturbojpeg: 1.5.0 -> 1.5.1

https://github.com/lilohuang/PyTurboJPEG/releases/tag/v1.5.1

SFrijters committed:
python3Packages.maestral: Remove redundant override and update homepage

SFrijters committed:
maestral-gui: 1.4.3 -> 1.4.4

dotlambda committed:
python3Packages.kombu: fix tests

mweinelt committed:
python3Packages.aiohttp: remove pytest-xdist

Fails reproducible on hydra like this:

```
        try:
            # set_wakeup_fd() raises ValueError if this is not the
            # main thread.  By calling it early we ensure that an
            # event loop running in another thread cannot add a signal
            # handler.
>           signal.set_wakeup_fd(self._csock.fileno())
E           ValueError: set_wakeup_fd only works in main thread of the main interpreter
```

And was remove from upstreams tests due to 10-15% error rate way back
in 2018.

mweinelt committed:
python3Packages.mwparserfromhell: fix tests; add missing check inputs

mweinelt committed:
python3Packages.matrix-nio: relax aiofiles constraint

mweinelt committed:
python3Packages.marisa-trie: disable broken tests

Some tests fail due hypothesis healthchecks, that complains about a
temporary directory not being cleaned between tests.

This is due to the hypothesis 6.x upgrade, while the library pins 2.0.0
from 2016.

mweinelt committed:
python3Packages.markdown-it-py: relax attrs contraint

mweinelt committed:
python3Packages.karton-dashboard: relax prometheus-client constraint

mweinelt committed:
python3Packages.junos-eznc: relax ncclient constraint

mweinelt committed:
python3Packages.hbmqtt: patch for websockets 9.0 compat

mweinelt committed:
python3Packages.flask-api: 2.0 -> 3.0.post1

mweinelt committed:
python3Packages.flask_migrate: 2.7.0 -> 3.0.1

mweinelt committed:
python3Packages.extractcode: disable failing tests

mweinelt committed:
python3Packages.enamlx: remove obsolete patch

mweinelt committed:
python3Packages.emv: relax click and pyscard constraint

mweinelt committed:
python3Packages.diagrams: relax jinja2 constraint

mweinelt committed:
pinnwand: relax click, sqlalchemy and docutils constraints

Signed-off-by: Martin Weinelt <hexa@darmstadt.ccc.de>

mweinelt committed:
steck: relax click constraint

mweinelt committed:
python3Packages.debugpy: fix tests with flask>=2.0

mweinelt committed:
python3Packages.python-miio: relax click constraint; fix cli

The CLI requires pyyaml which was previously only a checkInput.

mweinelt committed:
home-assistant: pin aiopvpc, pyatmo, pyatv, pysma

fabaff committed:
home-assistant-cli: 0.9.3 -> 0.9.4

mweinelt committed:
backblaze-b2: 2.4.0 -> 2.5.0

Relaxes the docutils constraint

mweinelt committed:
python3Packages.billiard: 3.6.3.0 -> 3.6.4.0

mweinelt committed:
python3Packages.shamir-mneomic: relax click constraint

dotlambda committed:
python3Packages.requests-cache: update dependencies, run tests

mweinelt committed:
python3Packages.sphinx-rtd-theme: fix build

Adds the missing docutils dependency and relaxes its constraint.
Migrates from pytest to pytestCheckHook and moves the CI environment
variable export into preBuild.

mweinelt committed:
python3Packages.anyio: 3.1.0 -> 3.2.0

Fix version string by including setuptools-scm and setting a pretend
version, so that we don't require other version hints.

sternenseemann committed:
haskell.compiler.ghc8104: patch for sphinx >= 4.0

With sphinx 4, interpreting the conf.py fails due to a decode
error: https://gitlab.haskell.org/ghc/ghc/-/issues/19962

The fix is an one line change which we have to backport from GHC master.
Thus ghcHEAD is not affected by this problem.

9.2 and 8.10.6 will most likely have a fix for this.

TODO: Patch 8.8.4 and 9.0.1

dotlambda committed:
python3Packages.jupyter_server: update dependencies

mweinelt committed:
python3Packages.jupyter_server: drop anyio version substitution

mweinelt committed:
python3Packages.httpcore: drop anyio version substitution

mweinelt committed:
python3Packages.psycopg2: disable when python<3.6, update license

dotlambda committed:
pythonPackages.dictpath: init at 0.1.3

dotlambda committed:
pythonPackages.openapi-core: init at 0.14.2

dotlambda committed:
python3Packages.nbclassic: 0.2.6 -> 0.3.1

SuperSandro2000 commented on pkgs/development/python-modules/billiard/default.nix:
```diff
@@ -2,22 +2,19 @@
-    sha256 = "0spssl3byzqsplra166d59jx8iqfxyzvcbx7vybkmwr5ck72a5yr";
+    sha256 = "0ismj2p8c66ykpss94rs0bfra5agxxmljz8r3gaq79r8valfb799";
   };
-  patches = [(fetchpatch {
```
Please remove the unused fetchpatch from inputs.

mweinelt replied:
Fixed in https://github.com/NixOS/nixpkgs/pull/127419/commits/7d9db0318371adb4f9770f83802dc15b233b731e, will be squashed into https://github.com/NixOS/nixpkgs/pull/127419/commits/4c229018efb2d2c5f20f10f0bceefdf565d5448a before merge.

dotlambda committed:
python3Packages.jupyterlab_server: update dependencies, fix tests

dotlambda committed:
python3Packages.jupyter-packaging: 0.8.2 -> 0.10.2

sternenseemann committed:
haskell.compiler.ghc884: patch for sphinx >= 4.0

With sphinx 4, interpreting the conf.py fails due to a decode
error: https://gitlab.haskell.org/ghc/ghc/-/issues/19962
The fix is an one line change which we have to backport from GHC
master.

9.2 and 8.10.6 will have a fix for this, GHC 9.0.1 and ghcHEAD
already have and GHC 8.10.4 has been patched in nixpkgs already.

dotlambda committed:
python37Packages.GitPython: fix build

mweinelt committed:
home-assistant: disable tesla component test

The tesla component test, via its authcaptureproxy dependency, depends
on typer, which in its current version is incompatible with click>=8.0.

mweinelt committed:
python3Packages.scipy: run tests in parallel

mweinelt committed:
whipper: add patch for ruamel_yaml 0.17 compat

dotlambda committed:
pythonPackages.pybind11: install cmake config again

dotlambda committed:
pythonPackages.pybind11: don't run scipy tests

Two tests out of 439 are not worth working around the cyclic dependency.

dotlambda committed:
pythonPackages.Kajiki: fix build

dotlambda committed:
python3Packages.gcsfs: 2021.05.0 -> 2021.06.0

dotlambda committed:
awscli: 1.19.88 -> 1.19.97

dotlambda committed:
python3Packages.pooch: update dependencies

dotlambda committed:
python3Packages.pyhiveapi: 0.4.2 -> 0.4.3

https://github.com/Pyhass/Pyhiveapi/releases/tag/v0.4.3

nixos-discourse commented:
This pull request has been mentioned on **NixOS Discourse**. There might be relevant details there:

https://discourse.nixos.org/t/policy-for-acceptable-number-of-failures-in-a-jobset/13851/1

bjornfor commented:
This broke `nix-build -A conan` because jinja2 was bumped from 2.x to 3.x.

dotlambda commented:
@bjornfor If conan is not compatible with jinja2 v3, then you can use `packageOverrides` in the expression for conan to use a different version.

bjornfor commented:
WIP fix for conan: https://github.com/NixOS/nixpkgs/pull/129137.
